src/dndbeyond/content-scripts/character.js

```diff
@@ -113,6 +113,14 @@ async function rollSkillCheck(paneClass) {
             roll_properties.d20 = `1d20min${min}`
         }
     }
+
+    if (character.hasRacialTrait("Deductive Intuition") && (skill_name == "Investigation" || skill_name == "Insight")){
```

Comment:
For the future, I'd do `&& ["Investigation", "Insight"].includes(skill_name)` as it's a nicer way to check for it and scales more easily.

Reply:
ah yeah, definitely does